database/rrdhost.c

```diff
@@ -447,6 +456,11 @@ RRDHOST *rrdhost_find_or_create(
 
         // update host tags
         rrdhost_init_tags(host, tags);
+#ifdef ENABLE_DBENGINE
+        if (host->rrd_memory_mode == RRD_MEMORY_MODE_DBENGINE) {
+            metalog_commit_update_host(host);
+        }
+#endif
```

Comment:
Can you add this in `rrdhost_create()` as well? We're missing a `HOST` command when streaming connects and it's a good opportunity to fix it here before we merge.

Reply:
Ok I will check it

This actually has an issue with streaming because it will do a `rrdhost_find_by_guid` and never actually try to update the info (via `rrdhost_find_or_create`) I am tweaking it to do so and make sure we will have all the info to write the metadata log entries (eg. labels)